### ISSUE TITLE
Something wrong with is_adult condition

### ISSUE THREAD

**Issue by mcepl** [JimmXinu/FanFicFare#482]
See this:
```sh
harmony@stitny$ fanficfare -o "is_adult=true" --update-epub --force The\ best\ laid\ plans-ao3_18862810.epub 
Updating The best laid plans-ao3_18862810.epub, URL: https://archiveofourown.org/works/18862810
..Please confirm you are an adult in your locale: (y/n)?
y
Please confirm you are an adult in your locale: (y/n)?
y
Traceback (most recent call last):
  File "/usr/bin/fanficfare", line 11, in <module>
    load_entry_point('FanFicFare==3.16.0', 'console_scripts', 'fanficfare')()
  File "/usr/lib/python3.8/site-packages/fanficfare/cli.py", line 305, in main
    do_download(url,
  File "/usr/lib/python3.8/site-packages/fanficfare/cli.py", line 490, in do_download
    output_filename = write_story(configuration, adapter, options.format,
  File "/usr/lib/python3.8/site-packages/fanficfare/cli.py", line 80, in write_story
    writer = writers.getWriter(writeformat, config, adapter)
  File "/usr/lib/python3.8/site-packages/fanficfare/writers/__init__.py", line 34, in getWriter
    return EpubWriter(config,story)
  File "/usr/lib/python3.8/site-packages/fanficfare/writers/writer_epub.py", line 55, in __init__
    BaseStoryWriter.__init__(self, config, story)
  File "/usr/lib/python3.8/site-packages/fanficfare/writers/base_writer.py", line 53, in __init__
    self.story = adapter.getStoryMetadataOnly() # only cache the metadata initially.
  File "/usr/lib/python3.8/site-packages/fanficfare/adapters/base_adapter.py", line 297, in getStoryMetadataOnly
    self.doExtractChapterUrlsAndMetadata(get_cover=get_cover)
  File "/usr/lib/python3.8/site-packages/fanficfare/adapters/base_adapter.py", line 381, in doExtractChapterUrlsAndMetadata
    return self.extractChapterUrlsAndMetadata()
  File "/usr/lib/python3.8/site-packages/fanficfare/adapters/adapter_archiveofourownorg.py", line 170, in extractChapterUrlsAndMetadata
    raise exceptions.AdultCheckRequired(self.url)
fanficfare.exceptions.AdultCheckRequired: Story requires confirmation of adult status: (https://archiveofourown.org/works/18862810)
harmony@stitny$
```
Not only I have provided ``-o "is_adult=true"``, but I have also ``is_adult:true`` in ``[defaults]`` section of [~/.fanficfare/personal.ini](https://github.com/JimmXinu/FanFicFare/files/4431998/personal.txt).

What’s wrong?

**Comment by JimmXinu** [comment]
Already fixed(#480).  Update to latest version.  

And please update to latest and review recent issues before opening new ones.  Thanks.

**Comment by mcepl** [comment]
Right, it is not possible to run ``fanficfare`` from the directory, one has to run ``python3 setup.py develop --user`` first.

This one has been really fixed. And I am sorry, I did look at the old issues (I worked for many years as a bugmaster so old habits die hard), but I haven’t really recognized that the ticket “Possible issue on AO3 for Calibre and CLI” has anything to do with the adult stories. I am sorry.
